examples/qualcomm/utils.py

```diff
@@ -438,14 +436,6 @@ def setup_common_args_and_variables():
         raise RuntimeError("Environment variable QNN_SDK_ROOT must be set")
     print(f"QNN_SDK_ROOT={os.getenv('QNN_SDK_ROOT')}")
 
-    if "LD_LIBRARY_PATH" not in os.environ:
```

Comment:
Is it for aihub flow only or is it general to the native flow?

Reply:
This change is general to native flow. User won't need to specify LD_LIBRARY_PATH anymore if artifacts and binaries are all stayed together.
AIHUB path is broken in 09/01 pytorch nightly, the custom op will trigger segfault in `torch.export.export`. I'm still looking for solution.

Reply:
I may miss some context, if libraries and binaries are not at the same path, we'd still need to specify the library path I assume?

Reply:
Yes. Or we could specify other rpath for the binaries if don't want to set LD_LIBRARY_PATH.